enterprise/signals.py

```diff
@@ -329,9 +328,8 @@ def create_enterprise_enrollment_receiver(sender, instance, **kwargs):     # pyl
     """
     if kwargs.get('created') and instance.user:
         user_id = instance.user.id
-        try:
-            ecu = EnterpriseCustomerUser.objects.get(user_id=user_id)
-        except ObjectDoesNotExist:
+        ecu = EnterpriseCustomerUser.objects.filter(user_id=user_id, active=True)
```

Comment:
`ecu` is here a `queryset`, and later on we are using it  as `ecu .id` at [line](https://github.com/edx/edx-enterprise/pull/1284/files#diff-b4b6533b246e5794f1a7051c1bc5b1848746e918dad92f5ea5c6ad09ada2fdfdR342). I think code will break here.

Reply:
Good catch! ... fixed